docs/python_gpu.rst

```diff
@@ -6,6 +6,9 @@ Quick-start examples
 
 As mentioned in the :ref:`Python GPU installation instructions <install-python-gpu>`, the easiest way to install the Python interface for cuFINUFFT is to run ``pip install cufinufft``.
 
+*Note*: The interface to cuFINUFFT has changed between versions 1.3 and 2.2.
+Please see :ref:`Migration to cuFINUFFT v2.2<cufinufft_migration>` for details.
+
 Assuming cuFINUFFT has been installed, we will now consider how to calculate a 1D type 1 transform.
 To manage the GPU and transfer to and from host and device, we will use the ``pycuda`` library.
```

Comment:
I think we need to add what isn't obvious to everyone: cuFINUFFT performs transforms on arrays living on the GPU. This is for speed, to allow combination with other GPU computations. To integrate with CPU computations, the user will need to perform H2D then D2H afterwards.

Reply:
Right. Will address this in #408 instead to avoid conflicts.

Reply:
See 6686076d0e6111a0f3eb84b6aa6f2e88d2773bf0.